tube/etl/indexers/base/parser.py

```diff
@@ -215,6 +215,8 @@ def get_type_of_item_for_list_type(self, node_label, src):
         return a.get(src)
 
     def get_prop_type(self, fn, src, node_label=None, index=None):
+        if src == "source_node":
+            return (str,)
```

Comment:
does this mean we assume that it's always a list of strings?

Reply:
this is always string. That is the expected notation for string type @paulineribeyre